docs/learn-about-neptune/ui.rst

```diff
@@ -0,0 +1,137 @@
+
+Neptune supports Chrome and Firefox, on all major operating systems.
+
+Log in to the app
+-----------------
+1. In a web browser, navigate to ``ui.neptune.ai`` or, in the case of an on-prem deployment, to your Neptune instance address.
+2. Click **Login**.
+
+.. image:: /_static/images/core-concepts/login.png
+   :target: /_static/images/core-concepts/login.png
+   :alt: Login screen
+   :width: 350
+
+
+3. Type in your credentials and click **Sign in**.
+
+Experiments View
+----------------
+
+
+The **Experiments** tab  displays all the experiments in a specific Project in table form.
+
+Filter Experiments
+^^^^^^^^^^^^^^^^^^
+You can perform the simplest filtering by typing into the search fields:
+
+   .. image:: /_static/images/core-concepts/search_fields.png
+      :target: /_static/images/core-concepts/search_fields.png
+      :alt: Experiment search
+
+
+You can also use the `Neptune Query Language <nql.html>`_ to filter experiments for more advanced criteria.
+
+Organize Experiments
+^^^^^^^^^^^^^^^^^^^^
+
+There are several ways to organize your experiments:
+
+- **Tags**: You can create tag(s), which you assign to experiments. Later, you can quickly filter by these tags.
+
+   .. image:: /_static/images/core-concepts/tag_chooser.png
+      :target: /_static/images/core-concepts/tag_chooser.png
+      :alt: Tag chooser
+      :width: 250
+
+- **Custom views**: You can create a custom view of the dashboard, for example, by filtering selected rows, columns and columns order, sorting and width. Then you can save the view and quickly return to it by selecting the desired view from the list of views.
+
+   .. image:: /_static/images/core-concepts/view_list.png
+      :target: /_static/images/core-concepts/view_list.png
+      :alt: View list
+      :width: 400
+
+- **Choosing columns**: Columns in the dashboard are configurable. All types of data logged to Neptune can be seen as columns in the dashboard, for example parameters, metrics or tags.
+
+- **Grouping**: You can group experiments by one or more column(s). The dashboard displays the selected columns, allowing you to make in-group and across-groups analysis of the experiments.
+
+   .. image:: /_static/images/core-concepts/group_by.png
+      :target: /_static/images/core-concepts/group_by.png
+      :alt: Group columns
+
+
+Compare experiments
+^^^^^^^^^^^^^^^^^^^
+Neptune lets you compare up to 10 experiments using in-depth analysis in the specialized view.
+Simply select experiments and click **Compare**:
+
+
+   .. image:: /_static/images/core-concepts/compare_experiments_select.png
+      :target: /_static/images/core-concepts/compare_experiments_select.png
+      :alt: Compare experiments table
+
+
+- **Overlaid charts**: In the comparison view, all metrics with the same name are placed on a single chart with one curve per experiment. The rich and customizable legend lets you display additional information.
+
+   .. image:: /_static/images/core-concepts/charts_legend.png
```

Comment:
This image doesn't show the actual legend, only a menu for choosing its elements. I'd much prefer showing the actual legend.

Reply:
Charts legend as in what can be seen on hover.oCharst
You need to first select a couple of metrics and/or parameters and then those will be seen on hover.